sidebars.js

```diff
@@ -84,23 +84,27 @@ module.exports = {
     },
     {
       type: 'category',
-      label: 'Staking',
+      label: 'Forging And Staking',
```

Comment:
The label can be updated, but it would better to keep the old file paths bc it's also how the URLs to the pages are generated and it wil break old links

Reply:
Done

Reply:
Hi @alsala , I had meant the opposite. The `label` can be changed to `Forging And Staking`, but the file names and paths to the markdowns will determine the URL to the pages on the docs site.

You can either change back the folder and file names, or for each file, add a `slug` to the front matter
https://docusaurus.io/docs/docs-introduction#docs-only-mode
For example, on the first file, add `slug: /horizen_eon/staking/what_is_staking`